### DISCUSSION TITLE
YT Revanced for Android 7 (LineageOS 14.1) with no history bug

### DISCUSSION THREAD

**Discussion by SkoDave** [kitadai31/revanced-patches-android6-7#82] [Q&A]
Hi, I'm going crazy to fix this issue, help please... I installed LineageOS 14.1 on Nvidia Shield Tablet K1 (there was the stock ROM but it became very slow and laggy and it was almost unusable) but I have problem with Revanced, I tried with some magisk module but it does not save YT history, I tried building with revanced manager 6-7 (on another device) but both root and no root crash when started. Could please help me or suggest a "ready" version to use?

**Comment by kitadai31** [comment]
https://github.com/kitadai31/revanced-patches-android6-7/wiki/How-to-build-(Rooted)
Follow this guide

**Reply by SkoDave** [reply]
I did follow the guide but the rooted version crashes at start, the non rooted version now starts (or probably it started before too, I did too many tries) but it doesnt save video progression or history.

**Reply by kitadai31** [reply]
Are you using brand account? (multi channels on one Google Account)
Then, go to Settings > ReVanced Settings > Miscellaneous > Spoof client
Change "iOS" to other spoofs

If not brand account, check your Adblock blocking "s.youtube.com" domain.
Example, AdAway is known to block this domain. Add it to white list.

**Reply by SkoDave** [reply]
Thank you for the tips, I tried both but sadly issue persist :(
